src/FlexItem.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module FlexItem
+ */
+class FlexItem extends React.Component {
+	render() {
+		const {
+			children,
+			className,
+			shrink,
+			growFactor,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			'flex-item',
+			{
+				'flex-item--shrink': shrink,
+				[`flex-item--${growFactor}`] : typeof growFactor === 'number',
+			},
+			className
+		);
+
+		return (
+			<div
+				className={classNames}
+				{...other}>
+					{children}
```

Comment:
there's an extra indent here, but it's a little masked because the `prop`s are also indented - I recommend putting the `>` on its own line at the same indent level as `<div` to make is clear where the tag ends and the children begin